### PR TITLE
Add missing iOS framework type

### PR DESCRIPTION
**Pull request by WeiYewT**
### Background ###

- add missing patch to track React framework type
- update Rokt-Widget version to 4.6
Fixes [(issue)]

### What Has Changed: ###

List out what has changed as a result of this PR.

### How Has This Been Tested? ###

Describe the tests that you ran to verify your changes. 

### Notes

Add any notes or extra information here that might be useful to the reviewer if applicable i.e. links to documentation/blogs or dashboards.

### Checklist: ###

- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have verified that CI completes successfully.
- [x] All insignificant commits have been squashed.

### CODE REVIEW

**Review by thomson-t — APPROVED** [review]

File: RoktSampleApp/ios/Podfile.lock [hunk 1]
```diff
@@ -351,10 +351,10 @@ PODS:
     - React-jsi (= 0.69.12)
     - React-logger (= 0.69.12)
     - React-perflogger (= 0.69.12)
-  - RNCCheckbox (0.5.17):
+  - RNCCheckbox (0.5.16):
```
Comment (lewis-rokt) [inline RoktSampleApp/ios/Podfile.lock:354]: Any particular reason to downgrade or did it just happen?

**Review by lewis-rokt — APPROVED** [review]

File: RoktSampleApp/ios/Podfile.lock (hunk 1 above)
Comment (WeiYewT) [reply RoktSampleApp/ios/Podfile.lock:354]: it just happened 👀

File: RoktSampleApp/package-lock.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
       "version": "0.0.1",
       "dependencies": {
         "@react-native-community/checkbox": "^0.5.12",
-        "@rokt/react-native-sdk": "file:../Rokt.Widget/",
+        "@rokt/react-native-sdk": "file:../Rokt.Widget/rokt-react-native-sdk-4.6.0.tgz",
```
Comment (lewis-rokt) [inline RoktSampleApp/package-lock.json:12]: Need to revert this

File: RoktSampleApp/package-lock.json [hunk 3]
```diff
@@ -4222,6 +4644,18 @@
       "resolved": "https://registry.npmjs.org/@react-native/polyfills/-/polyfills-2.0.0.tgz",
       "integrity": "sha512-K0aGNn1TjalKj+65D7ycc1//H9roAQ51GJVk5ZJQFb2teECGmzd86bYDC0aYdbRf7gtovescq4Zt6FR0tgXiHQ=="
     },
+    "node_modules/@rokt/react-native-sdk": {
```
Comment (lewis-rokt) [inline RoktSampleApp/package-lock.json:4647]: And remove this object

File: RoktSampleApp/package-lock.json (hunk 2 above)
Comment (WeiYewT) [reply RoktSampleApp/package-lock.json]: just realized the convention has been to not commit this file so ive removed it.

**Review by mirannakamura — APPROVED** [review]

**Review by danial-rokt — APPROVED** [review]

**Pull request merged**